### PR TITLE
Update public_suffix_list.dat vps8.de

### PR DESCRIPTION
**Pull request by chengdada123**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)
    - [Cloudflare](https://developers.cloudflare.com/learning-paths/get-started/add-domain-to-cf/add-site/)
    - [Letsencrypt](https://letsencrypt.org/docs/rate-limits/)
    - <UPDATE THIS LIST WITH YOUR LIMITATIONS! REMOVE ENTRIES WHICH DO NOT APPLY! REMOVE THIS LINE!>

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The submitter acknowledges that it is their responsibility to maintain the domains within their section. This includes removing names which are no longer used, retaining the _psl DNS entry, responding to e-mails to the supplied address. Failure to maintain entries may result in removal of individual entries or the entire section.

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

Organization Website: 

Reason for PSL Inclusion
====

Number of users this request is being made to serve:

DNS Verification via dig
=======

Results of Syntax Checker (`make test`)
=========

### CODE REVIEW

**Comment by chengdada123** [comment]
Description of Organization
Our organization provides free subdomains to developers for testing programs and software. We aim to support developers by offering reliable and easily accessible domain services for their projects.

Organization Website
https://vps8.de

Reason for PSL Inclusion
Inclusion in the Public Suffix List (PSL) is critical for our organization because it ensures that the subdomains we provide to developers are treated as separate, independent domains. This is important for several reasons:

Security: It helps prevent cookies from being shared across different subdomains, which enhances security and privacy for users testing their applications on our platform.
Development Flexibility: Developers can use these subdomains without worrying about interference from cookies and policies applied to other subdomains, making the testing environment more realistic and effective.
Email Deliverability: It ensures proper handling of emails sent from these subdomains, avoiding potential issues with spam filters and ensuring reliable communication during development and testing.
Number of users this request is being made to serve
This request is intended to serve an estimated maximum number of users, as our service is open to all developers globally who need a secure and reliable domain for their testing purposes.

**Comment by groundcat** [comment]
Concerns:

- The sorting is incorrect. As noted in the [guideline](https://github.com/publicsuffix/list/wiki/Guidelines), "for a new private domain section, keep the changeset sorted by Company/Product name." I recommend reviewing the guidelines thoroughly.
- Under the section "Number of users this request is being made to serve," your response might be too vague. Can you provide a more accurate estimation of the existing users of the domain? Is this a new service without any existing users yet? The domain seems to have been registered today. According to [CT](https://crt.sh/?q=vps8.de), 2 wildcard SSL certs were issued today. Subdomain discovery found no subdomains. Google SERP returns no results. The signup functionality at https://vps8.de is currently not working. Note:

> Projects that are smaller in scale or are temporary or seasonal in nature will likely be declined. Examples of this might be private-use, sandbox, test, lab, beta, or other exploratory nature changes or requests. It should be expected that despite whatever site or service referred a requestor to seek addition of their domain(s) to the list, projects not serving more than thousands of users are quite likely to be declined.

- You mentioned you are trying to work around the limits of Cloudflare and Let's Encrypt. Please consult the [guideline](https://github.com/publicsuffix/list/wiki/Guidelines) and then confirm if this is the case. If not, please remove these lines.
- Is DevSubDomains the company/organization name? On the https://vps8.de/ website, the support email is listed as `support@devsubdomains.com`, but the domain `devsubdomains.com` is not registered based on WHOIS.

Other comments:

- Unable to verify the expiration date of `vps8.de` due to the lack of expiration date in WHOIS records.
- A personal email address is used in comments.
- The TXT record is valid at `_psl.vps8.de`.

File: public_suffix_list.dat [hunk 1]
```diff
@@ -15843,4 +15843,8 @@ basicserver.io
 virtualserver.io
 enterprisecloud.nu
 
+//vps8 :http://vps8.de
+//Submitted by cheng <chengc512@gmail.com>
```
Comment (wdhdev) [inline public_suffix_list.dat:15847]:
```suggestion
// vps8 : http://vps8.de
// Submitted by cheng <chengc512@gmail.com>
```

Comment (wdhdev) [inline public_suffix_list.dat:15847]: Preferably using a non-personal email address is best here.

**Review by wdhdev — COMMENTED** [review]
You need to fix the sorting of your entry to be alphabetical. Please also refer to my comments below.

**Comment by chengdada123** [comment]
============================================================================
Testsuite summary for libpsl 0.21.5
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================

**Comment by wdhdev** [comment]
You still need to fix the sorting of your domain, please read the guidelines: https://github.com/publicsuffix/list/wiki/Guidelines

**Comment by chengdada123** [comment]
Thank you very much, it has been modified

**Comment by simon-friedberger** [comment]
* [x] Expiration (Note: Must STAY >2y at all times)
  * [ ] Expiration date for vps8.de unavailable ⚠️
* [x] DNS _psl entries (Note: Must STAY in place)
* [x] Tests pass
* [x] Sorting
* [ ] Reasoning/Organization description
* [x] Non-personal email address

**Comment by chengdada123** [comment]
image
We are a community of programming enthusiasts with multiple active Telegram groups, each with thousands of members. We are dedicated to providing high-quality services to these enthusiasts. We sincerely hope that you will approve our application. We pledge to strictly adhere to your terms and ensure that the domain remains active and continues to provide services. Thank you very much!

**Comment by wdhdev** [comment]
> We are a community of programming enthusiasts with multiple active Telegram groups, each with thousands of members. We are dedicated to providing high-quality services to these enthusiasts. We sincerely hope that you will approve our application. We pledge to strictly adhere to your terms and ensure that the domain remains active and continues to provide services. Thank you very much!

That still doesn't explain why you need the domain on the PSL. Do you need it for cookie separation, URL highlighting, etc.

**Comment by chengdada123** [comment]
Thank you for your reply!
We are a community of programming enthusiasts with multiple active Telegram groups, each with thousands of members. We are dedicated to providing high-quality services to these enthusiasts. We sincerely hope that you will approve our application.

To better serve our community, we need our domain to be included in the Public Suffix List (PSL). This will help us achieve the following goals:

Cookie Separation: Ensure that cookies for each subdomain are stored independently, enhancing user privacy and security.
URL Highlighting: Make our subdomains more recognizable in the browser address bar, improving user experience.
Email Handling: Ensure that emails from these subdomains are handled correctly, avoiding issues with spam filters.
We pledge to strictly adhere to your terms and ensure that the domain remains active and continues to provide services. Thank you very much!

**Comment by chengdada123** [comment]
This is not AI!
I just wanted to word it better.
I hope this is accurate.
I don't know how to do it better. Thanks!

**Comment by chengdada123** [comment]
I revised the wording, thank you for your criticism!

**Comment by simon-friedberger** [comment]
We need some form of evidence that there are a lot of users for this.

> Projects that are smaller in scale or are temporary or seasonal in nature will likely be declined. Examples of this might be private-use, sandbox, test, lab, beta, or other exploratory nature changes or requets. It should be expected that despite whatever site or service referred a requestor to seek addition of their domain(s) to the list, projects not serving more then thousands of users are quite likely to be declined.

The common approach is to provide the service and come back when it is publicly visible that there is a huge amount of users. But, if you want to provide other evidence we will consider that as well.

**Comment by dnsguru** [comment]
this reminds me of #1562

**Comment by groundcat** [comment]
Pinging @chengdada123, please respond if you would like to proceed or abort, so the volunteers can either continue or close this efficiently. If you are setting up a subdomain registration service as shown on your homepage, the suggestion is to close this PR and come back when you have a substantial user base, unless you have other verifiable evidence suggesting an existing large user base (more than thousands).
请及时回复是否继续或放弃这项PR，以便志愿者开展工作。如果您计划开设某种二级域名注册服务，我们建议先关闭此PR，并在未来用户数量足够庞大（至少超过数千活跃用户）之后，再考虑重新提交PSL请求，除非您能提供其他可验证的证据表明已经存在大量用户。谢谢。

**Comment by chengdada123** [comment]
I really appreciate the hard work of the volunteers, I'd like to come back once I have a large user base. Thanks

**Pull request closed without merging**